### PR TITLE
gitea-actions-runner: unstable-2023-02-08 -> unstable-2023-03-18

### PR DESCRIPTION
**Pull request by LongerHV**
###### Description of changes

Updates gitea-actions-runner.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/continuous-integration/gitea-actions-runner/default.nix [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
     owner = "gitea";
     repo = "act_runner";
-    rev = "990cf93c7136669408eb1832cd05df3ad4dd81b3";
-    sha256 = "1ysp7g199dzh1zpxxhki88pn96qghln7a5g8zfjip9173q1rgiyb";
+    rev = "v${version}";
+    sha256 = "sha256-2a5a9AcRNsjWSlUtGw4fF/cixVNoEGOg+KXkQBELui8=";
   };
 
-  vendorSha256 = "0a3q7rsk37dc6v3vnqaywkimaqvyjmkrwljhcjcnswsdfcgng62b";
+  vendorSha256 = "sha256-+d5KG841nnKhR/Rl/pm+wX6WIchBsaLBQxRaGYR5D+8=";
```
Comment (Kranzes) [inline pkgs/development/tools/continuous-integration/gitea-actions-runner/default.nix:16]:
```suggestion

  ldflags = [
    "-s"
    "-w"
    "-X gitea.com/gitea/act_runner/cmd.version=${version}"
  ];

```

**Comment by bennyandresen** [comment]
Build it. Registered a runner with gitea and also ran an example workflow. LGTM and could be merged.

The version in the first line of the registration is 0.1.5, but that seems like an upstream issue. [^1]

[^1] https://gitea.com/gitea/act_runner#register

**Comment by Kranzes** [comment]
> Built it. Registered a runner with gitea and also ran an example workflow. LGTM and could be merged.
> 
> The version in the first line of the registration is 0.1.5, but that seems like an upstream issue. [^1]
> 
> [^1] https://gitea.com/gitea/act_runner#register

waiting for ofborg.

**Review by SuperSandro2000 — APPROVED** [review]
please squash merge to fix the commit message

**Review by techknowlogick — CHANGES_REQUESTED** [review]
sorry for blocknig this, the tag was solely so that goreleaser would build the nightly binaries, and I should add a note that it shouldn't be treated as a release. Could you update to the current latest release instead?

**Comment by LongerHV** [comment]
> sorry for blocknig this, the tag was solely so that goreleaser would build the nightly binaries, and I should add a note that it shouldn't be treated as a release. Could you update to the current latest release instead?

Oh, I'll change it back to unstable if that is the case.

**Comment by techknowlogick** [comment]
> Oh, I'll change it back to unstable if that is the case.

Thanks, and again I'm sorry about that.

**Review by techknowlogick — APPROVED** [review]
Thanks so much!!

**Comment by LongerHV** [comment]
There was some `ofborg-internal-error`. I have no idea how to troubleshoot this, so I have rebased and we will see...

**Comment by bennyandresen** [comment]
I couldn't built this package using `nix-review` without the `--eval local` (it defaults to "ofborg" it seems.)

Works fine and now the version in the first line of the register command is "unstable-2023-03-18". To me this looks good.

```bash
λ nix-review pr 222217 --eval local
$ git -c fetch.prune=false fetch --no-tags --force https://github.com/NixOS/nixpkgs master:refs/nixpkgs-review/0 pull/222217/head:refs/nixpkgs-review/1
$ git worktree add /home/benny/.cache/nixpkgs-review/pr-222217-7/nixpkgs 3d50e8355068c139197870753528954887656ccf
Preparing worktree (detached HEAD 3d50e835506)
Updating files: 100% (34056/34056), done.
HEAD is now at 3d50e835506 Merge pull request #222287 from zendo/update/musicfox
$ nix-env --option system x86_64-linux -f /home/benny/.cache/nixpkgs-review/pr-222217-7/nixpkgs -qaP --xml --out-path --show-trace --no-allow-import-from-derivation
$ git merge --no-commit --no-ff daf33aca55f06d5f8cab69b904ad36b4916c0416
Automatic merge went well; stopped before committing as requested
$ nix-env --option system x86_64-linux -f /home/benny/.cache/nixpkgs-review/pr-222217-7/nixpkgs -qaP --xml --out-path --show-trace --no-allow-import-from-derivation --meta
1 package updated:
gitea-actions-runner (2023-02-08 → 2023-03-18)

$ nix --experimental-features nix-command build --no-link --keep-going --no-allow-import-from-derivation --option build-use-sandbox relaxed -f /home/benny/.cache/nixpkgs-review/pr-
222217-7/build.nix

Link to currently reviewing PR:
https://github.com/NixOS/nixpkgs/pull/222217

1 package built:
gitea-actions-runner
```

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**